### ISSUE TITLE
Attestation Validation tests are not implemented

### ISSUE THREAD

**Issue by paulhauner** [sigp/lighthouse#97]
## Description

Attestation validation has been implemented, however the tests are not.

The tests rely upon the new block structure (#96) which has not been implemented.

## Steps to resolve

Once #96 is complete, come back an implement tests.

**Comment by paulhauner** [comment]
This functionality is removed in #163

**Comment by paulhauner** [comment]
Closing in favor of #182
